code/controllers/subsystem/fastprocess.dm

```diff
@@ -2,13 +2,9 @@ var/datum/subsystem/fastprocess/SSfastprocess
 
 /datum/subsystem/fastprocess
 	name = "Fast Process"
-	priority = 12
-	wait = 1
-	dynamic_wait = 1
-	dwait_upper = 10
-	dwait_buffer = 0
-	dwait_lower = 1
-	dwait_delta = 6
+	priority = 25
+	flags = SS_BACKGROUND|SS_POST_FIRE_TIMING|SS_NO_INIT
+	wait = 4
```

Comment:
It is now... I bumped that down to 2ds.

It COULD go faster, but i don't want it eating up too much of the ticks.